cli/tools/coverage/mod.rs

```diff
@@ -619,10 +623,21 @@ pub async fn cover_files(
     return Err(generic_error("No matching coverage profiles found"));
   }
 
+  // file:///var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/deno-cli-testM9DwRS/npm/registry.npmjs.org
+  // file:///private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/deno-cli-testM9DwRS/npm/registry.npmjs.org
+  let cli_options = Arc::new(CliOptions::from_flags(flags.clone())?);
   let ps = ProcState::build(flags).await?;
 
+  let npm_domain = ps.npm_api.base_url().domain().unwrap(); // e.g. registry.npmjs.org
+  let deno_dir = cli_options.resolve_deno_dir()?;
+  let mut npm_folder_path = deno_dir.npm_folder_path();
+  npm_folder_path.push(npm_domain);
+
+  let npm_folder_filepath =
+    Url::from_file_path(npm_folder_path.to_str().unwrap()).unwrap();
   let script_coverages = collect_coverages(coverage_flags.files)?;
   let script_coverages = filter_coverages(
+    npm_folder_filepath.as_str(),
```

Comment:
Instead of all the above, the directory of where the npm files are found can be found on `ps.npm_resolver.fs_resolver.root_dir_url()`. We need to expose it on `npm_resolver` as something like `ps.npm_resolver.root_dir_url()`. Then we can provide the url to this function.